File: EventFlowAPI/EventFlowAPI.Logic/DTO/Statistics/ResponseDto/EventPassStatistics.cs
namespace EventFlowAPI.Logic.DTO.Statistics.ResponseDto
{
    public class EventPassStatistics
    {
        public int AllBougthEventPassesCount { get; set; }
        public int AllRenewedEventPassesCount { get; set; }
        public int AllCanceledEventPassesCount { get; set; }
        public double TotalEventPassesIncome { get; set; }
        public Dictionary<string, double> EventPassTypeDict { get; set; } = [];
        public Dictionary<string, double> MostProfitableEventsPassTypeDict { get; set; } = [];
    }
}

File: EventFlowClient/src/models/response_models/stats/EventPassStatistics.ts
export type EventPassStatistics = {
  allBoughtEventPassesCount: number;
  allRenewedEventPassesCount: number;
  allCanceledEventPassesCount: number;
  totalEventPassesIncome: number;
  eventPassTypeDict: Record<string, number>;
  mostProfitableEventsPassTypeDict: Record<string, number>;
};
